package.json

```diff
@@ -18,6 +18,7 @@
     "e2e:enterprise:dev": "./e2e/start-and-run-suite enterprise dev",
     "e2e:enterprise:debug": "./e2e/start-and-run-suite enterprise debug",
     "e2e:playwright": "yarn playwright test",
+    "e2e:playwright:server": "./e2e/plugin-e2e/start-and-run-suite",
```

Comment:
Should we add a note about this command to the [docs](https://github.com/grafana/grafana/blob/main/contribute/style-guides/e2e-plugins.md#commands) as well?

Reply:
Good point! Added to the e2e-plugins.md docs and to the developer-guide.